python-crud-google-spreadsheet/deploy.sh

```diff
@@ -0,0 +1,6 @@
+./bob/build.sh 3
```

Comment:
Not sure if this is the best flow. Realistically Bob should only be run once after dependencies are changed. I understand it's needed the first time but afterwards there will most likely just be code changes.

Reply:
Removing and adding to README